src/main/app/claims/models/directions-questionnaire/directionsQuestionnaire.ts

```diff
@@ -67,6 +56,40 @@ export namespace DirectionsQuestionnaire {
     }
   }
 
+
+  export function toHearingLocation(directionsQuestionnaire: DirectionsQuestionnaireDraft): HearingLocation {
```

Comment:
maybe we don't need to export function, i don't think we can or will use it anywhere else...

Reply:
Have reverted. @KiranVarma02